flow360/cloud/requests.py

```diff
@@ -28,6 +28,22 @@ class Config:  # pylint: disable=too-few-public-methods
         allow_population_by_field_name = True
 
 
+class NewSurfaceMeshRequest(Flow360Requests):
+    """request for new surface mesh"""
+
+    name: str = pd.Field(alias="name")
+    stem: str = pd.Field(alias="stem")
```

Comment:
what is stem?

Reply:
stem comes from https://en.cppreference.com/w/cpp/filesystem/path/stem